sklearn/utils/setup.py

```diff
@@ -54,6 +55,25 @@ def configuration(parent_package='', top_path=None):
                          include_dirs=[numpy.get_include()],
                          libraries=libraries)
 
+    # generate files from a template
+    pyx_templates = ['sklearn/utils/seq_dataset.pyx.tp',
+                     'sklearn/utils/seq_dataset.pxd.tp']
+
+    for pyxfiles in pyx_templates:
+        assert pyxfiles.endswith(('.pyx.tp', '.pxd.tp'))
+        outfile = pyxfiles[:-3]
+        # if .pxi.in is not updated, no need to output .pxi
```

Comment:
Edit comment to match code. Maybe precise that this a good idea for cythonization ?